packages/esm-patient-registration-app/src/patient-registration/section/section-wrapper.component.tsx

```diff
@@ -23,7 +23,7 @@ export const SectionWrapper = ({ sectionDefinition, index }: SectionWrapperProps
    * t('relationshipsSection', 'Relationships')
    */
   return (
-    <div id={sectionDefinition.id}>
+    <div id={sectionDefinition.id} style={{ scrollMarginTop: '100px' }}>
```

Comment:
We usually use the `rem` instead of `px`. The correct value will be `4rem`, `3rem` is the primary navigation and the `1rem` will be the spacing from the primary navigation.
Thanks!